config/theme.yml

```diff
@@ -10,6 +10,7 @@ meta:
   compatibility:
     from: 9.0.0
     to: ~
+    framework: 'bootstrap-v4.0.0-alpha.5'
```

Comment:
Discussion with @jolelievre: what should be the content of
- 'bootstrap-v4.0.0-alpha.5'
- 'v4'
- 'bootstrap-v4.0.0'
- ...

Reply:
@matks 

- `bootstrap-v4.0.0-alpha.5`
- `bootstrap-v5.3.3`
- `tailwind-v3.4.10`